docs/tasks/configure-pod-container/configure-pod-disruption-budget.md

```diff
@@ -49,6 +49,23 @@ specified in the budget, thus bringing the number of available pods from the
 collection below the specified size. The budget can only protect against
 voluntary evictions, not all causes of unavailability.
 
+You can find an example of a pod disruption budget defined below. It matches pods with the label 
+`app: zookeeper`.
+
+```yaml
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  labels:
```

Comment:
I would suggest removing this so that people don't get confused and think this part might be doing the matching. (I know we use this pattern in other examples, though, and I always wonder if people get confused...)